analysis/parameter_tuning.py

```diff
@@ -139,6 +138,11 @@ def _find_candidate_parameters(
     l0_bounds = linf_bounds = None
     max_sum_per_partition_bounds = min_sum_per_partition_bounds = None
 
+    if calculate_sum_per_partition_param:
+        assert not parameters_to_tune.min_sum_per_partition, "Tuning of min_sum_per_partition is not supported yet"
```

Comment:
Please put this check in [_check_tune_args](https://github.com/OpenMined/PipelineDP/blob/72faa2c1758316425e285a9ca09ce13191025b45/analysis/parameter_tuning.py#L313)

Reply:
added, wdyt? afaiu options.parameters_to_tune.min_sum_per_partition should never be true because we don't support it in any cases.

Reply:
Yes, agree that tuning min_sum_per_partition should be false for now